ext/opentelemetry-ext-system-metrics/README.rst

```diff
@@ -0,0 +1,23 @@
+OpenTelemetry System Metrics Instrumentation
+============================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-ext-system-metrics.svg
+   :target: https://pypi.org/project/opentelemetry-ext-system-metrics/
+
+Instrumentation to collect system performance metrics.
+
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-ext-system-metrics
+
+
+References
+----------
+* `OpenTelemetry Project <https://opentelemetry.io/>`_
+
```

Comment:
Nit: Link to readthedocs?, that btw, the entry in the docs is missing.

Reply:
Fixed.